### ISSUE TITLE
clearer table of contents

### ISSUE THREAD

**Issue by aghaynes** [strengejacke/ggeffects#559]
Hey!
I just noticed that the table of contents seems to have gotten a bit messed up in the introduction_comparisons_2 vignette...
![image](https://github.com/user-attachments/assets/af9eac8d-4dba-4c43-9d9e-a652ddb38664)

It seems that the callout is "stealing" the content of the sections that come after it...
https://github.com/strengejacke/ggeffects/blob/9bd782add940e183ecc49ac58e494831e1f2fbcd/vignettes/introduction_comparisons_2.Rmd#L65
![image](https://github.com/user-attachments/assets/3106a8bd-7ab5-4a21-a950-0b67f3195bf8)

Perhaps moving the callout above the "Hypothesis testing for slopes of numeric predictors" title would work better? :smile:

**Comment by strengejacke** [comment]
No, I think the behaviour is that subsection / subheadings only appear when you scroll to the related section, to avoid to lengthy tables of content. This has changed recently in the package-down website features, so this should be intentional. :-)

**Comment by aghaynes** [comment]
Sure, but the headings are not within the summary, are they? That's how it currently shows, as if you had done 

```
# Hypothesis testing
...
# Summary of important topics
...
## is the linear trend significant
...
## is the difference linear trend significant
...
## etc
```

What you intended, I guess, is the following, which also makes more sense to me (the summary is just that, it's not a section itself per se, or certainly not a top one)...

```
# Hypothesis testing
...
## is the linear trend significant
...
## is the difference linear trend significant
...
## etc
```

(perhaps with the summary as a first second level section)

**Comment by strengejacke** [comment]
I see. It's because the header of the summary box is declared as `<h1>`.
